src/backend/aspen/api/views/phylo_runs.py

```diff
@@ -182,3 +185,56 @@ async def kick_off_phylo_run(
     )
 
     return PhyloRunResponseSchema.from_orm(workflow)
+
+
+async def get_editable_phylorun_by_id(db, group_id, run_id, user):
+    aliased(PhyloRun)
+    query = (
+        sa.select(PhyloRun)
+        .filter(
+            sa.and_(
+                PhyloRun.group == user.group,  # This is an access control check!
+                PhyloRun.group_id
+                == group_id,  # This makes sure the group ID in our path is accurate
+                PhyloRun.id == run_id,
+            )
+        )
+        .options(joinedload(PhyloRun.outputs))
+    )
+    results = await db.execute(query)
+    try:
+        run = results.scalars().unique().one()
+    except NoResultFound:
+        raise ex.NotFoundException("phylo run not found")
+    if run.workflow_status == WorkflowStatusType.STARTED:
+        raise ex.NotFoundException("Can't modify an in-progress phylo run")
+    return run
+
+
+@router.get("/")
+async def list_runs(
+    request: Request,
+    db: AsyncSession = Depends(get_db),
+    settings: Settings = Depends(get_settings),
+    user: User = Depends(get_auth_user),
+) -> bool:
+    return False
+
+
+@router.delete("/{group_id}/{item_id}")
```

Comment:
If the `item_id` here is the phylo run PK -- I'm pretty sure it is -- I don't see the point of also putting the `group_id` in the route. We can identify the PhyloRun by the PK, and we know the `group_id` by inspecting the PhyloRun. It's structurally similar to the `delete_sample` route, but if we're going to move that over to PKs in the near future, I think we should drop the group_id aspect over there too when that happens.